fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayBanner.kt

```diff
@@ -198,23 +237,87 @@ private fun NormalTabsTabIcon(normalTabCount: Int) {
     }
 }
 
+/**
+ * Banner displayed in multi select mode.
+ *
+ * @param bannerItems List of icons to be displayed in the banner.
+ * @param menuItems List of menu items to be displayed in the menu at the end of the banner. Null if
+ * menu should not be displayed.
+ */
 @Composable
-private fun MultiSelectBanner() {
-    Box(
+private fun MultiSelectBanner(
+    selectedTabs: Int,
+    bannerItems: List<BannerItem> = emptyList(),
+    menuItems: List<MenuItem>? = null,
+) {
+    var showMenu by remember { mutableStateOf(false) }
+
+    Row(
         modifier = Modifier
             .fillMaxWidth()
-            .height(80.dp)
+            .height(88.dp)
             .background(color = FirefoxTheme.colors.layerAccent),
-        contentAlignment = Alignment.Center,
     ) {
+        IconButton(
+            onClick = {},
```

Comment:
I opened https://bugzilla.mozilla.org/show_bug.cgi?id=1826994 to handle the close button, since I realized we didn't have a ticket for it yet.